inc/wpseo-non-ajax-functions.php

```diff
@@ -247,6 +247,9 @@ function wpseo_admin_bar_menu() {
 
 	// @todo: add links to bulk title and bulk description edit pages.
 	if ( $user_is_admin_or_networkadmin ) {
+
+		$advanced_settings = wpseo_advanced_settings_enabled( $options );
```

Comment:
The first time I saw `$advanced_settings` I thought is contains an array with advanced settings. I think `has_advanced_settings_enabled` or `advanced_settings_enabled` would be better.